### PR TITLE
refactor(frontend): remove custom class prop from RoundedIcon

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

The prop iconStyleClass seemed unnecessary for just opacity. So we substitute with a direct boolean prop (reference https://github.com/dfinity/oisy-wallet/pull/3537#discussion_r1842195829).

### CODE REVIEW

File: src/frontend/src/lib/components/ui/RoundedIcon.svelte [hunk 1]
```diff
@@ -2,11 +2,11 @@
 	import type { ComponentType } from 'svelte';
 
 	export let icon: ComponentType;
-	export let iconStyleClass = '';
+	export let withOpacity = false;
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/ui/RoundedIcon.svelte:5]: We can also call the property just "opacity"

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/ui/RoundedIcon.svelte (hunk 1 above)
Comment (AntonioVentilii) [reply src/frontend/src/lib/components/ui/RoundedIcon.svelte:5]: changed

**Comment by AntonioVentilii** [comment]
[OISY-130]

**Pull request merged**